### PR TITLE
Add Forensic-night CTF writeups

### PR DESCRIPTION
**Pull request by aenniw**
Signed-off-by: Martin Mihálek <aenniw@gmail.com>

### CODE REVIEW

File: forensic-night.cesnet.cz/proprietarni-data/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#### Challenge:
+
+Analyzujte datový soubor proprietární aplikace, kterou už nelze sehnat. Porozumět způsobu ukládání dat by se docela hodilo, ať už je v souboru uloženo cokoliv. [file.dat.gz](./file.dat.gz ':ignore')
+
+---
+
+#### Solution:
+
+```sql
+select Char from data where PartOf = (select ID from ids where Description = 'flag') order by Position;
```
Comment (fabrymarko) [inline forensic-night.cesnet.cz/proprietarni-data/README.md]: this looks familiar too

File: forensic-night.cesnet.cz/ssh-backdoor/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#### Challenge:
+
+Náš systémový administrátor má tušení, že použitý SSH server obsahuje backdoor. Můžete jej v přiloženém kódu najít? Když se vám to podaří, ověřte jej v praxi a získejte přístup na tento server: `ssh ctf@forensic-night.cesnet.cz -p 20002` [dropbear-2019.78-backdoored.tar.gz](./dropbear-2019.78-backdoored.tar.gz ':ignore')
+
+---
+
+#### Solution:
+
+```bash
+ll -lt ./dropbear-2019.78/ | head -n 5
```
Comment (fabrymarko) [inline forensic-night.cesnet.cz/ssh-backdoor/README.md]: ??? how did you find this?

File: forensic-night.cesnet.cz/PDF-I/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+---
+
+#### Solution:
+
+```bash
+gunzip pdf1.pdf.gz
+pdf-parser.py  -o 4 ./pdf1.pdf -d pdf-dump.bin
+for c in $( cat ./pdf-dump.bin | grep -i secret | tr '<>' ' ' | awk '{ print $2 }' ); do \
+    echo $c | base32 -d 2>/dev/null  | base32 -d; echo; \
```
[the remark this answers is not in the document]
Comment (aenniw) [reply forensic-night.cesnet.cz/PDF-I/README.md]: readabilty as there are two flags at the end and without it its on the same line

File: forensic-night.cesnet.cz/cerna-a-bila/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+```python
+#!/usr/bin/env python3
+
+from PIL import Image
+
+codes = []
+byte = ""
+
+with Image.open('black_and_white.png', 'r') as img:
```
[the remark this answers is not in the document]
Comment (aenniw) [reply forensic-night.cesnet.cz/cerna-a-bila/README.md]: Yes :smile:

File: forensic-night.cesnet.cz/proprietarni-data/README.md (hunk 1 above)
Comment (aenniw) [reply forensic-night.cesnet.cz/proprietarni-data/README.md]: Yes :smile:

File: forensic-night.cesnet.cz/ssh-backdoor/README.md (hunk 2 above)
Comment (aenniw) [reply forensic-night.cesnet.cz/ssh-backdoor/README.md]: in gz are `.gitignore` and `.hg` I wanted to check last commit if any... so in the end I ended listing most recent files a manually inspected them, also lookup of `passwd` and lot of manual inspection would reveal it ...

File: forensic-night.cesnet.cz/bezpecny-domov/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+#### Solution:
+
+```console
+(Hex(Asc(Mid(DDVV, 1, 1))) <> "50")                         => chr(int('0x50', 16))     => 'P'
+(Asc(Mid(DDVV, 2, 1)) <> 97)                                => chr(97)                  => 'a'
+(Asc(Mid(DDVV, 3, 1)) <> 118)                               => chr(118)                 => 'v'
+(Hex(Asc(Mid(DDVV, 4, 1))) <> "65")                         => chr(int('0x65', 16))     => 'e'
+(Asc(Mid(DDVV, 5, 1)) <> 108)                               => chr(108)                 => 'l'
+(Oct(Asc(Mid(DDVV, 6, 1))) <> 137)                          => chr(int('137', 8))       => ' '
```
[the remark this answers is not in the document]
Comment (aenniw) [reply forensic-night.cesnet.cz/bezpecny-domov/README.md]: fixed

File: forensic-night.cesnet.cz/ssh-backdoor/README.md (hunk 2 above)
Comment (fabrymarko) [reply forensic-night.cesnet.cz/ssh-backdoor/README.md]: Nicu.

**Pull request merged**